portal/src/graphql/portal/AuthenticatorConfigurationScreen.tsx

```diff
@@ -420,9 +479,20 @@ const AuthenticationAuthenticatorSettingsContent: React.FC<AuthenticationAuthent
           <WidgetTitle>
             <FormattedMessage id="AuthenticatorConfigurationScreen.primary-authenticators.title" />
           </WidgetTitle>
+          {hasPrimaryFeatureDisabled && (
+            <MessageBar>
+              <FormattedMessage
+                id="FeatureConfig.disabled"
```

Comment:
This message here would be better if it reads `The grayed out features are not available for your project plan. Upgrade your project plan.`.